src/App.tsx

```diff
@@ -15,36 +17,109 @@
   'Garlic',
 ];
 
-export const App: React.FC = () => {
+const SORT_FIELD_ALPHABET: string = 'alphabet';
+const SORT_FIELD_LENGTH: string = 'length';
+const SORT_FIELD_RESET: string = '';
+
+type Props = {
+  sortField: string;
+  setSortField: () => string;
+  isReverse: string;
+  setReverse: () => string;
```

Comment:
[CODE KNOWLEDGE] - Similarly, `setReverse` should be a function that takes a boolean argument to update the state, e.g., `(value: boolean) => void`.